container_files/tutorials/Loading data from HTTP servers or online file storage services.ipynb

```diff
@@ -0,0 +1,156 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Loading data from an HTTP server\n",
+    "\n",
+    "MLDB gives users full control over where and how data is persisted. MLDB handles multiple protocol for URLs (see [Files and URLs](../../../../doc/#builtin/Url.md.html)). In this tutorial, we provide examples to load files via:\n",
+    "* <code> file:// </code> for a file inside an MLDB container (see the [Loading Data Tutorial](../../../../ipy/notebooks/_tutorials/_latest/Loading%20Data%20Tutorial.ipynb) for an example)\n",
+    "* <code> http:// </code> or <code> https://</code> for a file accessible on a HTTP server on the public internet or a private intranet "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The notebook cells below use `pymldb`'s `Connection` class to make [REST API](../../../../doc/#builtin/WorkingWithRest.md.html) calls. You can check out the [Using `pymldb` Tutorial](../../../../doc/nblink.html#_tutorials/Using pymldb Tutorial) for more details."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from pymldb import Connection\n",
+    "mldb = Connection()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading data with http:// or https://\n",
+    "MLDB makes it very easy to load with the public web. A powerful feature of MLDB allows us to load publicly avaialbe data directly from a web address. The datasets below are from the [Stanford Network Analysis Project (SNAP)](http://snap.stanford.edu/index.html). Here, we have chosen to load [Facebook's Social Circles](http://snap.stanford.edu/data/egonets-Facebook.html) dataset. In our first example, we will be [`importing a text file`](../../../../doc/#builtin/procedures/importtextprocedure.md.html)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<Response [201]>\n"
+     ]
+    }
+   ],
+   "source": [
+    "dataUrl = \"http://snap.stanford.edu/data/facebook_combined.txt.gz\"\n",
```

Comment:
J'ajouterais une ligne qui mentionne que le fichier est .gz: Notice the file we will load is gzipped. Not only are we loading the file from a remote location, but we are also decompressing it transparently as it's being downloaded.